docs/essentials/widget_language.md

```diff
@@ -420,15 +419,15 @@ Text {
 > **Tips**
 >
 > - `TypographyTheme` configures the text style used in the application as a part of `Theme`. Click [`Introduction to Theme[wip]`] to learn more.
-> - `BuildCtx` provides many contexts for the application of the `widget!`.
+> - **`BuildCtx` provides many contexts for the application of the `widget!`.**(I wasn't able to figure out what this could mean. There is only `theme` and `windowCtx` )
```

Comment:
You're right, there are only theme and windowCtx. We might add some in the future.